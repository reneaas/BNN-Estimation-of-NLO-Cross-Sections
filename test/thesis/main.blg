This is BibTeX, Version 0.99d (TeX Live 2021)
Capacity: max_strings=200000, hash_size=200000, hash_prime=170003
The top-level auxiliary file: main.aux
The style file: JHEP.bst
Database file #1: bibliography.bib
You've used 67 entries,
            2188 wiz_defined-function locations,
            1051 strings with 18828 characters,
and the built_in function-call counts, 21024 in all, are:
= -- 2265
> -- 674
< -- 0
+ -- 210
- -- 143
* -- 1593
:= -- 2863
add.period$ -- 83
call.type$ -- 67
change.case$ -- 55
chr.to.int$ -- 0
cite$ -- 67
duplicate$ -- 1073
empty$ -- 1918
format.name$ -- 143
if$ -- 5490
int.to.chr$ -- 0
int.to.str$ -- 67
missing$ -- 235
newline$ -- 219
num.names$ -- 67
pop$ -- 181
preamble$ -- 1
purify$ -- 0
quote$ -- 0
skip$ -- 831
stack$ -- 0
substring$ -- 1327
swap$ -- 531
text.length$ -- 0
text.prefix$ -- 0
top$ -- 0
type$ -- 0
warning$ -- 0
while$ -- 171
width$ -- 69
write$ -- 681
