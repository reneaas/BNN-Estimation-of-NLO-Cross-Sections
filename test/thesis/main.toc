\babel@toc {UKenglish}{}
\contentsline {chapter}{Introduction}{1}{chapter*.4}%
\contentsline {chapter}{\numberline {1}Introduction to Quantum Field Theory}{3}{chapter.1}%
\contentsline {section}{\numberline {1.1}Basic Construction and Lagrangians}{3}{section.1.1}%
\contentsline {section}{\numberline {1.2}Canonical Quantization}{7}{section.1.2}%
\contentsline {section}{\numberline {1.3}The Path Integral Formalism}{9}{section.1.3}%
\contentsline {subsection}{\numberline {1.3.1}Path Integral in Quantum Mechanics}{10}{subsection.1.3.1}%
\contentsline {subsection}{\numberline {1.3.2}Path Integrals in Quantum Field Theory}{13}{subsection.1.3.2}%
\contentsline {subsection}{\numberline {1.3.3}Quantization of Scalar Field Theory}{16}{subsection.1.3.3}%
\contentsline {subsection}{\numberline {1.3.4}Quantization of Fermionic Fields}{17}{subsection.1.3.4}%
\contentsline {subsection}{\numberline {1.3.5}Quantization of Abelian Gauge Fields}{19}{subsection.1.3.5}%
\contentsline {section}{\numberline {1.4}Perturbation Theory in Quantum Field Theory}{21}{section.1.4}%
\contentsline {subsection}{\numberline {1.4.1}Perturbation Theory from Path Integrals}{22}{subsection.1.4.1}%
\contentsline {subsection}{\numberline {1.4.2}LSZ-Reduction Formula and Cross Section}{24}{subsection.1.4.2}%
\contentsline {section}{\numberline {1.5}Regularization and Renormalization}{30}{section.1.5}%
\contentsline {subsection}{\numberline {1.5.1}Dimensional Regularization and Feynman Paramerization}{31}{subsection.1.5.1}%
\contentsline {subsection}{\numberline {1.5.2}Renormalized Perturbation Theory}{34}{subsection.1.5.2}%
\contentsline {subsection}{\numberline {1.5.3}The Callan-Symanzik Equation}{38}{subsection.1.5.3}%
\contentsline {chapter}{\numberline {2}Geometry of Gauge Theories and Wilson Lines}{41}{chapter.2}%
\contentsline {section}{\numberline {2.1}Mathematical Concepts in Gauge Theories}{41}{section.2.1}%
\contentsline {subsection}{\numberline {2.1.1}Basics in Differential Geometry}{42}{subsection.2.1.1}%
\contentsline {subsection}{\numberline {2.1.2}Gauge Fields as Connections on Principal Fibre Bundles}{48}{subsection.2.1.2}%
\contentsline {subsection}{\numberline {2.1.3}Curvature and Field Strength}{54}{subsection.2.1.3}%
\contentsline {section}{\numberline {2.2}Yang-Mills Theories from Wilson Lines}{57}{section.2.2}%
\contentsline {subsection}{\numberline {2.2.1}Covariant Derivative and Wilson Lines}{57}{subsection.2.2.1}%
\contentsline {subsection}{\numberline {2.2.2}Field Strength Tensor and Wilson Loops}{59}{subsection.2.2.2}%
\contentsline {subsection}{\numberline {2.2.3}Quantization of Yang-Mills Theories}{61}{subsection.2.2.3}%
\contentsline {section}{\numberline {2.3}Wilson Line Properties}{63}{section.2.3}%
\contentsline {chapter}{\numberline {3}Perturbative Quantum Chromodynamics}{69}{chapter.3}%
\contentsline {section}{\numberline {3.1}Field Theoretical Description}{69}{section.3.1}%
\contentsline {section}{\numberline {3.2}Deep Inelastic Scattering}{74}{section.3.2}%
\contentsline {subsection}{\numberline {3.2.1}DIS and the Parton Model}{74}{subsection.3.2.1}%
\contentsline {subsection}{\numberline {3.2.2}Collinear Factorization in QCD}{79}{subsection.3.2.2}%
\contentsline {subsection}{\numberline {3.2.3}Operator Definition for PDFs}{87}{subsection.3.2.3}%
\contentsline {subsection}{\numberline {3.2.4}Gauge Invariant Parton Distributions}{89}{subsection.3.2.4}%
\contentsline {subsection}{\numberline {3.2.5}Parton-in-Parton Distributions}{91}{subsection.3.2.5}%
\contentsline {section}{\numberline {3.3}Drell-Yan Cross Section in QCD}{93}{section.3.3}%
\contentsline {subsection}{\numberline {3.3.1}LO Drell-Yan Cross Section}{93}{subsection.3.3.1}%
\contentsline {subsection}{\numberline {3.3.2}NLO Drell-Yan Cross Section}{95}{subsection.3.3.2}%
\contentsline {subsection}{\numberline {3.3.3}Renormalization of NLO cross section}{99}{subsection.3.3.3}%
\contentsline {subsection}{\numberline {3.3.4}Numerical Evaluation of the Hadronic Drell-Yan Cross Section}{101}{subsection.3.3.4}%
\contentsline {chapter}{\numberline {4}Resummation using Wilson Lines}{103}{chapter.4}%
\contentsline {section}{\numberline {4.1}Exponentiation}{103}{section.4.1}%
\contentsline {section}{\numberline {4.2}Mellin Space Factorization}{107}{section.4.2}%
\contentsline {subsection}{\numberline {4.2.1}Phase Space Factorization}{107}{subsection.4.2.1}%
\contentsline {subsection}{\numberline {4.2.2}Hadronic Cross Section in Mellin Space}{108}{subsection.4.2.2}%
\contentsline {section}{\numberline {4.3}Threshold Factorization}{110}{section.4.3}%
\contentsline {section}{\numberline {4.4}Factorization of Soft Gluons}{111}{section.4.4}%
\contentsline {section}{\numberline {4.5}Renormalization of Wilson Lines}{113}{section.4.5}%
\contentsline {subsection}{\numberline {4.5.1}One-Loop Cusp Anomalous Dimension}{115}{subsection.4.5.1}%
\contentsline {section}{\numberline {4.6}Exponentiation of Parton-In-Parton Distributions}{118}{section.4.6}%
\contentsline {subsection}{\numberline {4.6.1}Hard Virtual Gluons}{119}{subsection.4.6.1}%
\contentsline {section}{\numberline {4.7}The Eikonal Cross Section}{120}{section.4.7}%
\contentsline {section}{\numberline {4.8}Logarithmic Corrections in Drell-Yan}{124}{section.4.8}%
\contentsline {section}{\numberline {4.9}Hadronic Cross Section and Inverse Mellin}{125}{section.4.9}%
\contentsline {subsection}{\numberline {4.9.1}The Inverse Mellin Transform}{125}{subsection.4.9.1}%
\contentsline {subsection}{\numberline {4.9.2}Hadronic Cross Section in $x$-space}{126}{subsection.4.9.2}%
\contentsline {chapter}{Conclusion}{129}{chapter*.54}%
\contentsline {chapter}{Appendices}{131}{section*.55}%
\contentsline {chapter}{Appendix A}{133}{appendix*.56}%
\contentsline {section}{\numberline {A.1}Light-Cone Coordinates}{133}{section.Alph0.1}%
\contentsline {chapter}{Appendix B}{135}{appendix*.57}%
\contentsline {section}{\numberline {B.1}Dirac Gamma Matrices}{135}{section.Alph0.1}%
\contentsline {chapter}{Appendix C}{137}{appendix*.58}%
\contentsline {section}{\numberline {C.1}Plus Distributions}{137}{section.Alph0.1}%
\contentsline {chapter}{Appendix D}{139}{appendix*.59}%
\contentsline {section}{\numberline {D.1}The Mellin Transform}{139}{section.Alph0.1}%
\contentsline {subsection}{\numberline {D.1.1}Mellin Transforms of Functions}{140}{subsection.Alph0.1.1}%
