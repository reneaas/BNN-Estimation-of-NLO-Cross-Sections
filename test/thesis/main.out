\BOOKMARK [0][-]{chapter*.4}{\376\377\000I\000n\000t\000r\000o\000d\000u\000c\000t\000i\000o\000n}{}% 1
\BOOKMARK [0][-]{chapter.1}{\376\377\000I\000n\000t\000r\000o\000d\000u\000c\000t\000i\000o\000n\000\040\000t\000o\000\040\000Q\000u\000a\000n\000t\000u\000m\000\040\000F\000i\000e\000l\000d\000\040\000T\000h\000e\000o\000r\000y}{}% 2
\BOOKMARK [1][-]{section.1.1}{\376\377\000B\000a\000s\000i\000c\000\040\000C\000o\000n\000s\000t\000r\000u\000c\000t\000i\000o\000n\000\040\000a\000n\000d\000\040\000L\000a\000g\000r\000a\000n\000g\000i\000a\000n\000s}{chapter.1}% 3
\BOOKMARK [1][-]{section.1.2}{\376\377\000C\000a\000n\000o\000n\000i\000c\000a\000l\000\040\000Q\000u\000a\000n\000t\000i\000z\000a\000t\000i\000o\000n}{chapter.1}% 4
\BOOKMARK [1][-]{section.1.3}{\376\377\000T\000h\000e\000\040\000P\000a\000t\000h\000\040\000I\000n\000t\000e\000g\000r\000a\000l\000\040\000F\000o\000r\000m\000a\000l\000i\000s\000m}{chapter.1}% 5
\BOOKMARK [2][-]{subsection.1.3.1}{\376\377\000P\000a\000t\000h\000\040\000I\000n\000t\000e\000g\000r\000a\000l\000\040\000i\000n\000\040\000Q\000u\000a\000n\000t\000u\000m\000\040\000M\000e\000c\000h\000a\000n\000i\000c\000s}{section.1.3}% 6
\BOOKMARK [2][-]{subsection.1.3.2}{\376\377\000P\000a\000t\000h\000\040\000I\000n\000t\000e\000g\000r\000a\000l\000s\000\040\000i\000n\000\040\000Q\000u\000a\000n\000t\000u\000m\000\040\000F\000i\000e\000l\000d\000\040\000T\000h\000e\000o\000r\000y}{section.1.3}% 7
\BOOKMARK [2][-]{subsection.1.3.3}{\376\377\000Q\000u\000a\000n\000t\000i\000z\000a\000t\000i\000o\000n\000\040\000o\000f\000\040\000S\000c\000a\000l\000a\000r\000\040\000F\000i\000e\000l\000d\000\040\000T\000h\000e\000o\000r\000y}{section.1.3}% 8
\BOOKMARK [2][-]{subsection.1.3.4}{\376\377\000Q\000u\000a\000n\000t\000i\000z\000a\000t\000i\000o\000n\000\040\000o\000f\000\040\000F\000e\000r\000m\000i\000o\000n\000i\000c\000\040\000F\000i\000e\000l\000d\000s}{section.1.3}% 9
\BOOKMARK [2][-]{subsection.1.3.5}{\376\377\000Q\000u\000a\000n\000t\000i\000z\000a\000t\000i\000o\000n\000\040\000o\000f\000\040\000A\000b\000e\000l\000i\000a\000n\000\040\000G\000a\000u\000g\000e\000\040\000F\000i\000e\000l\000d\000s}{section.1.3}% 10
\BOOKMARK [1][-]{section.1.4}{\376\377\000P\000e\000r\000t\000u\000r\000b\000a\000t\000i\000o\000n\000\040\000T\000h\000e\000o\000r\000y\000\040\000i\000n\000\040\000Q\000u\000a\000n\000t\000u\000m\000\040\000F\000i\000e\000l\000d\000\040\000T\000h\000e\000o\000r\000y}{chapter.1}% 11
\BOOKMARK [2][-]{subsection.1.4.1}{\376\377\000P\000e\000r\000t\000u\000r\000b\000a\000t\000i\000o\000n\000\040\000T\000h\000e\000o\000r\000y\000\040\000f\000r\000o\000m\000\040\000P\000a\000t\000h\000\040\000I\000n\000t\000e\000g\000r\000a\000l\000s}{section.1.4}% 12
\BOOKMARK [2][-]{subsection.1.4.2}{\376\377\000L\000S\000Z\000-\000R\000e\000d\000u\000c\000t\000i\000o\000n\000\040\000F\000o\000r\000m\000u\000l\000a\000\040\000a\000n\000d\000\040\000C\000r\000o\000s\000s\000\040\000S\000e\000c\000t\000i\000o\000n}{section.1.4}% 13
\BOOKMARK [1][-]{section.1.5}{\376\377\000R\000e\000g\000u\000l\000a\000r\000i\000z\000a\000t\000i\000o\000n\000\040\000a\000n\000d\000\040\000R\000e\000n\000o\000r\000m\000a\000l\000i\000z\000a\000t\000i\000o\000n}{chapter.1}% 14
\BOOKMARK [2][-]{subsection.1.5.1}{\376\377\000D\000i\000m\000e\000n\000s\000i\000o\000n\000a\000l\000\040\000R\000e\000g\000u\000l\000a\000r\000i\000z\000a\000t\000i\000o\000n\000\040\000a\000n\000d\000\040\000F\000e\000y\000n\000m\000a\000n\000\040\000P\000a\000r\000a\000m\000e\000r\000i\000z\000a\000t\000i\000o\000n}{section.1.5}% 15
\BOOKMARK [2][-]{subsection.1.5.2}{\376\377\000R\000e\000n\000o\000r\000m\000a\000l\000i\000z\000e\000d\000\040\000P\000e\000r\000t\000u\000r\000b\000a\000t\000i\000o\000n\000\040\000T\000h\000e\000o\000r\000y}{section.1.5}% 16
\BOOKMARK [2][-]{subsection.1.5.3}{\376\377\000T\000h\000e\000\040\000C\000a\000l\000l\000a\000n\000-\000S\000y\000m\000a\000n\000z\000i\000k\000\040\000E\000q\000u\000a\000t\000i\000o\000n}{section.1.5}% 17
\BOOKMARK [0][-]{chapter.2}{\376\377\000G\000e\000o\000m\000e\000t\000r\000y\000\040\000o\000f\000\040\000G\000a\000u\000g\000e\000\040\000T\000h\000e\000o\000r\000i\000e\000s\000\040\000a\000n\000d\000\040\000W\000i\000l\000s\000o\000n\000\040\000L\000i\000n\000e\000s}{}% 18
\BOOKMARK [1][-]{section.2.1}{\376\377\000M\000a\000t\000h\000e\000m\000a\000t\000i\000c\000a\000l\000\040\000C\000o\000n\000c\000e\000p\000t\000s\000\040\000i\000n\000\040\000G\000a\000u\000g\000e\000\040\000T\000h\000e\000o\000r\000i\000e\000s}{chapter.2}% 19
\BOOKMARK [2][-]{subsection.2.1.1}{\376\377\000B\000a\000s\000i\000c\000s\000\040\000i\000n\000\040\000D\000i\000f\000f\000e\000r\000e\000n\000t\000i\000a\000l\000\040\000G\000e\000o\000m\000e\000t\000r\000y}{section.2.1}% 20
\BOOKMARK [2][-]{subsection.2.1.2}{\376\377\000G\000a\000u\000g\000e\000\040\000F\000i\000e\000l\000d\000s\000\040\000a\000s\000\040\000C\000o\000n\000n\000e\000c\000t\000i\000o\000n\000s\000\040\000o\000n\000\040\000P\000r\000i\000n\000c\000i\000p\000a\000l\000\040\000F\000i\000b\000r\000e\000\040\000B\000u\000n\000d\000l\000e\000s}{section.2.1}% 21
\BOOKMARK [2][-]{subsection.2.1.3}{\376\377\000C\000u\000r\000v\000a\000t\000u\000r\000e\000\040\000a\000n\000d\000\040\000F\000i\000e\000l\000d\000\040\000S\000t\000r\000e\000n\000g\000t\000h}{section.2.1}% 22
\BOOKMARK [1][-]{section.2.2}{\376\377\000Y\000a\000n\000g\000-\000M\000i\000l\000l\000s\000\040\000T\000h\000e\000o\000r\000i\000e\000s\000\040\000f\000r\000o\000m\000\040\000W\000i\000l\000s\000o\000n\000\040\000L\000i\000n\000e\000s}{chapter.2}% 23
\BOOKMARK [2][-]{subsection.2.2.1}{\376\377\000C\000o\000v\000a\000r\000i\000a\000n\000t\000\040\000D\000e\000r\000i\000v\000a\000t\000i\000v\000e\000\040\000a\000n\000d\000\040\000W\000i\000l\000s\000o\000n\000\040\000L\000i\000n\000e\000s}{section.2.2}% 24
\BOOKMARK [2][-]{subsection.2.2.2}{\376\377\000F\000i\000e\000l\000d\000\040\000S\000t\000r\000e\000n\000g\000t\000h\000\040\000T\000e\000n\000s\000o\000r\000\040\000a\000n\000d\000\040\000W\000i\000l\000s\000o\000n\000\040\000L\000o\000o\000p\000s}{section.2.2}% 25
\BOOKMARK [2][-]{subsection.2.2.3}{\376\377\000Q\000u\000a\000n\000t\000i\000z\000a\000t\000i\000o\000n\000\040\000o\000f\000\040\000Y\000a\000n\000g\000-\000M\000i\000l\000l\000s\000\040\000T\000h\000e\000o\000r\000i\000e\000s}{section.2.2}% 26
\BOOKMARK [1][-]{section.2.3}{\376\377\000W\000i\000l\000s\000o\000n\000\040\000L\000i\000n\000e\000\040\000P\000r\000o\000p\000e\000r\000t\000i\000e\000s}{chapter.2}% 27
\BOOKMARK [0][-]{chapter.3}{\376\377\000P\000e\000r\000t\000u\000r\000b\000a\000t\000i\000v\000e\000\040\000Q\000u\000a\000n\000t\000u\000m\000\040\000C\000h\000r\000o\000m\000o\000d\000y\000n\000a\000m\000i\000c\000s}{}% 28
\BOOKMARK [1][-]{section.3.1}{\376\377\000F\000i\000e\000l\000d\000\040\000T\000h\000e\000o\000r\000e\000t\000i\000c\000a\000l\000\040\000D\000e\000s\000c\000r\000i\000p\000t\000i\000o\000n}{chapter.3}% 29
\BOOKMARK [1][-]{section.3.2}{\376\377\000D\000e\000e\000p\000\040\000I\000n\000e\000l\000a\000s\000t\000i\000c\000\040\000S\000c\000a\000t\000t\000e\000r\000i\000n\000g}{chapter.3}% 30
\BOOKMARK [2][-]{subsection.3.2.1}{\376\377\000D\000I\000S\000\040\000a\000n\000d\000\040\000t\000h\000e\000\040\000P\000a\000r\000t\000o\000n\000\040\000M\000o\000d\000e\000l}{section.3.2}% 31
\BOOKMARK [2][-]{subsection.3.2.2}{\376\377\000C\000o\000l\000l\000i\000n\000e\000a\000r\000\040\000F\000a\000c\000t\000o\000r\000i\000z\000a\000t\000i\000o\000n\000\040\000i\000n\000\040\000Q\000C\000D}{section.3.2}% 32
\BOOKMARK [2][-]{subsection.3.2.3}{\376\377\000O\000p\000e\000r\000a\000t\000o\000r\000\040\000D\000e\000f\000i\000n\000i\000t\000i\000o\000n\000\040\000f\000o\000r\000\040\000P\000D\000F\000s}{section.3.2}% 33
\BOOKMARK [2][-]{subsection.3.2.4}{\376\377\000G\000a\000u\000g\000e\000\040\000I\000n\000v\000a\000r\000i\000a\000n\000t\000\040\000P\000a\000r\000t\000o\000n\000\040\000D\000i\000s\000t\000r\000i\000b\000u\000t\000i\000o\000n\000s}{section.3.2}% 34
\BOOKMARK [2][-]{subsection.3.2.5}{\376\377\000P\000a\000r\000t\000o\000n\000-\000i\000n\000-\000P\000a\000r\000t\000o\000n\000\040\000D\000i\000s\000t\000r\000i\000b\000u\000t\000i\000o\000n\000s}{section.3.2}% 35
\BOOKMARK [1][-]{section.3.3}{\376\377\000D\000r\000e\000l\000l\000-\000Y\000a\000n\000\040\000C\000r\000o\000s\000s\000\040\000S\000e\000c\000t\000i\000o\000n\000\040\000i\000n\000\040\000Q\000C\000D}{chapter.3}% 36
\BOOKMARK [2][-]{subsection.3.3.1}{\376\377\000L\000O\000\040\000D\000r\000e\000l\000l\000-\000Y\000a\000n\000\040\000C\000r\000o\000s\000s\000\040\000S\000e\000c\000t\000i\000o\000n}{section.3.3}% 37
\BOOKMARK [2][-]{subsection.3.3.2}{\376\377\000N\000L\000O\000\040\000D\000r\000e\000l\000l\000-\000Y\000a\000n\000\040\000C\000r\000o\000s\000s\000\040\000S\000e\000c\000t\000i\000o\000n}{section.3.3}% 38
\BOOKMARK [2][-]{subsection.3.3.3}{\376\377\000R\000e\000n\000o\000r\000m\000a\000l\000i\000z\000a\000t\000i\000o\000n\000\040\000o\000f\000\040\000N\000L\000O\000\040\000c\000r\000o\000s\000s\000\040\000s\000e\000c\000t\000i\000o\000n}{section.3.3}% 39
\BOOKMARK [2][-]{subsection.3.3.4}{\376\377\000N\000u\000m\000e\000r\000i\000c\000a\000l\000\040\000E\000v\000a\000l\000u\000a\000t\000i\000o\000n\000\040\000o\000f\000\040\000t\000h\000e\000\040\000H\000a\000d\000r\000o\000n\000i\000c\000\040\000D\000r\000e\000l\000l\000-\000Y\000a\000n\000\040\000C\000r\000o\000s\000s\000\040\000S\000e\000c\000t\000i\000o\000n}{section.3.3}% 40
\BOOKMARK [0][-]{chapter.4}{\376\377\000R\000e\000s\000u\000m\000m\000a\000t\000i\000o\000n\000\040\000u\000s\000i\000n\000g\000\040\000W\000i\000l\000s\000o\000n\000\040\000L\000i\000n\000e\000s}{}% 41
\BOOKMARK [1][-]{section.4.1}{\376\377\000E\000x\000p\000o\000n\000e\000n\000t\000i\000a\000t\000i\000o\000n}{chapter.4}% 42
\BOOKMARK [1][-]{section.4.2}{\376\377\000M\000e\000l\000l\000i\000n\000\040\000S\000p\000a\000c\000e\000\040\000F\000a\000c\000t\000o\000r\000i\000z\000a\000t\000i\000o\000n}{chapter.4}% 43
\BOOKMARK [2][-]{subsection.4.2.1}{\376\377\000P\000h\000a\000s\000e\000\040\000S\000p\000a\000c\000e\000\040\000F\000a\000c\000t\000o\000r\000i\000z\000a\000t\000i\000o\000n}{section.4.2}% 44
\BOOKMARK [2][-]{subsection.4.2.2}{\376\377\000H\000a\000d\000r\000o\000n\000i\000c\000\040\000C\000r\000o\000s\000s\000\040\000S\000e\000c\000t\000i\000o\000n\000\040\000i\000n\000\040\000M\000e\000l\000l\000i\000n\000\040\000S\000p\000a\000c\000e}{section.4.2}% 45
\BOOKMARK [1][-]{section.4.3}{\376\377\000T\000h\000r\000e\000s\000h\000o\000l\000d\000\040\000F\000a\000c\000t\000o\000r\000i\000z\000a\000t\000i\000o\000n}{chapter.4}% 46
\BOOKMARK [1][-]{section.4.4}{\376\377\000F\000a\000c\000t\000o\000r\000i\000z\000a\000t\000i\000o\000n\000\040\000o\000f\000\040\000S\000o\000f\000t\000\040\000G\000l\000u\000o\000n\000s}{chapter.4}% 47
\BOOKMARK [1][-]{section.4.5}{\376\377\000R\000e\000n\000o\000r\000m\000a\000l\000i\000z\000a\000t\000i\000o\000n\000\040\000o\000f\000\040\000W\000i\000l\000s\000o\000n\000\040\000L\000i\000n\000e\000s}{chapter.4}% 48
\BOOKMARK [2][-]{subsection.4.5.1}{\376\377\000O\000n\000e\000-\000L\000o\000o\000p\000\040\000C\000u\000s\000p\000\040\000A\000n\000o\000m\000a\000l\000o\000u\000s\000\040\000D\000i\000m\000e\000n\000s\000i\000o\000n}{section.4.5}% 49
\BOOKMARK [1][-]{section.4.6}{\376\377\000E\000x\000p\000o\000n\000e\000n\000t\000i\000a\000t\000i\000o\000n\000\040\000o\000f\000\040\000P\000a\000r\000t\000o\000n\000-\000I\000n\000-\000P\000a\000r\000t\000o\000n\000\040\000D\000i\000s\000t\000r\000i\000b\000u\000t\000i\000o\000n\000s}{chapter.4}% 50
\BOOKMARK [2][-]{subsection.4.6.1}{\376\377\000H\000a\000r\000d\000\040\000V\000i\000r\000t\000u\000a\000l\000\040\000G\000l\000u\000o\000n\000s}{section.4.6}% 51
\BOOKMARK [1][-]{section.4.7}{\376\377\000T\000h\000e\000\040\000E\000i\000k\000o\000n\000a\000l\000\040\000C\000r\000o\000s\000s\000\040\000S\000e\000c\000t\000i\000o\000n}{chapter.4}% 52
\BOOKMARK [1][-]{section.4.8}{\376\377\000L\000o\000g\000a\000r\000i\000t\000h\000m\000i\000c\000\040\000C\000o\000r\000r\000e\000c\000t\000i\000o\000n\000s\000\040\000i\000n\000\040\000D\000r\000e\000l\000l\000-\000Y\000a\000n}{chapter.4}% 53
\BOOKMARK [1][-]{section.4.9}{\376\377\000H\000a\000d\000r\000o\000n\000i\000c\000\040\000C\000r\000o\000s\000s\000\040\000S\000e\000c\000t\000i\000o\000n\000\040\000a\000n\000d\000\040\000I\000n\000v\000e\000r\000s\000e\000\040\000M\000e\000l\000l\000i\000n}{chapter.4}% 54
\BOOKMARK [2][-]{subsection.4.9.1}{\376\377\000T\000h\000e\000\040\000I\000n\000v\000e\000r\000s\000e\000\040\000M\000e\000l\000l\000i\000n\000\040\000T\000r\000a\000n\000s\000f\000o\000r\000m}{section.4.9}% 55
\BOOKMARK [2][-]{subsection.4.9.2}{\376\377\000H\000a\000d\000r\000o\000n\000i\000c\000\040\000C\000r\000o\000s\000s\000\040\000S\000e\000c\000t\000i\000o\000n\000\040\000i\000n\000\040\000x\000-\000s\000p\000a\000c\000e}{section.4.9}% 56
\BOOKMARK [0][-]{chapter*.54}{\376\377\000C\000o\000n\000c\000l\000u\000s\000i\000o\000n}{}% 57
\BOOKMARK [0][-]{section*.55}{\376\377\000A\000p\000p\000e\000n\000d\000i\000c\000e\000s}{}% 58
\BOOKMARK [0][-]{appendix*.56}{\376\377\000A\000p\000p\000e\000n\000d\000i\000x\000\040\000A}{}% 59
\BOOKMARK [1][-]{section.Alph0.1}{\376\377\000L\000i\000g\000h\000t\000-\000C\000o\000n\000e\000\040\000C\000o\000o\000r\000d\000i\000n\000a\000t\000e\000s}{appendix*.56}% 60
\BOOKMARK [0][-]{appendix*.57}{\376\377\000A\000p\000p\000e\000n\000d\000i\000x\000\040\000B}{}% 61
\BOOKMARK [1][-]{section.Alph0.1}{\376\377\000D\000i\000r\000a\000c\000\040\000G\000a\000m\000m\000a\000\040\000M\000a\000t\000r\000i\000c\000e\000s}{appendix*.57}% 62
\BOOKMARK [0][-]{appendix*.58}{\376\377\000A\000p\000p\000e\000n\000d\000i\000x\000\040\000C}{}% 63
\BOOKMARK [1][-]{section.Alph0.1}{\376\377\000P\000l\000u\000s\000\040\000D\000i\000s\000t\000r\000i\000b\000u\000t\000i\000o\000n\000s}{appendix*.58}% 64
\BOOKMARK [0][-]{appendix*.59}{\376\377\000A\000p\000p\000e\000n\000d\000i\000x\000\040\000D}{}% 65
\BOOKMARK [1][-]{section.Alph0.1}{\376\377\000T\000h\000e\000\040\000M\000e\000l\000l\000i\000n\000\040\000T\000r\000a\000n\000s\000f\000o\000r\000m}{appendix*.59}% 66
\BOOKMARK [2][-]{subsection.Alph0.1.1}{\376\377\000M\000e\000l\000l\000i\000n\000\040\000T\000r\000a\000n\000s\000f\000o\000r\000m\000s\000\040\000o\000f\000\040\000F\000u\000n\000c\000t\000i\000o\000n\000s}{section.Alph0.1}% 67
